common/common.cpp

```diff
@@ -13,6 +14,26 @@ std::vector<int> make_random_uniform_binary(size_t size) {
   return out;
 }
 
+uint32_t make_random() {
+  std::random_device rd;
+  std::mt19937 gen(rd());
+  std::uniform_int_distribution<int> dist(1, 1000);
+  return dist(gen);
+}
+
+std::vector<uint32_t> make_unique_random(size_t size) {
+  std::random_device rd;
+  std::mt19937 gen(rd());
+  std::uniform_int_distribution<int> dist(1, std::min((long) size * 10, (long) ((uint32_t) -1)));
```

Comment:
`(long) ((uint32_t) -1)` looks a bit weird, did you want to generate a max uint32 value? If so, please use std numeric

Reply:
Also, what does the magic number 10 stand for?

Reply:
This is Daniil's implementation. Since he was already using unique keys, I didn't change his code.
First, I think size * 10 is stands for having keys values comparable to the size of the table. 
"(uint32)" replaced with std::numeric_limits<uint32_t>::max()